src/pages/bribersDashboard.tsx

```diff
@@ -6,7 +6,10 @@ import RoundSelector from "components/RoundSelector";
 
 const Dashboard: NextPage = () => {
   const { requestedRound, requestRound } = useGlobalContext();
-  const data = trpc.dashboard.list.useQuery({ round: requestedRound }).data?.board;
+  const data = trpc.dashboard.list.useQuery(
```

Comment:
this will not work, as there is no more "data", that was from one dashboard only, now we have 2. When merge, please do not delete new variables.